widget/widget.css

```diff
@@ -3,6 +3,15 @@
  * Licensed under the MIT License.
  */
 
+ .dev-container {
+    box-sizing: border-box;
+    color: #010;
+    font-size: 14px;
+    line-height: 20px;
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+ }
+
```

Comment:
So I did a visual comparison between what the widget looks like in the dev container vs Jupyter, and I still see differences.

Attaching two images. First one is within Jupyter and second one is within the dev window:

<img width="631" alt="Screen Shot 2020-10-14 at 11 09 05 PM" src="https://user-images.githubusercontent.com/680818/96084454-f5a21480-0e73-11eb-8a1a-db342d683d21.png">

<img width="624" alt="Screen Shot 2020-10-14 at 11 21 36 PM" src="https://user-images.githubusercontent.com/680818/96084511-0f435c00-0e74-11eb-8970-bd87158e096a.png">

Notice how some of the spacing is missing?

Reply:
Thanks for the screenshot. I wasn't sure exactly what else was missing. I'll see if I can fix that.